code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -49,7 +49,7 @@
 
 /obj/item/weapon/pinpointer/examine(mob/user)
 	..(user)
-	for(var/obj/machinery/nuclearbomb/bomb in world)
+	for(var/obj/machinery/nuclearbomb/bomb in machines)

```

Comment:
Nuclear bombs are not kept in this list; only machines that use power and/or process are in this.